client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -308,40 +267,21 @@ export class GoogleAnalyticsForm extends Component {
 const mapStateToProps = state => {
 	const site = getSelectedSite( state );
 	const siteId = getSelectedSiteId( state );
-	const siteSlug = getSelectedSiteSlug( state );
 	const isGoogleAnalyticsEligible =
 		site && site.plan && ( hasBusinessPlan( site.plan ) || isJetpackPremium( site.plan ) );
-	const jetpackManagementUrl = getSiteOption( state, siteId, 'admin_url' );
 	const jetpackModuleActive = isJetpackModuleActive( state, siteId, 'google-analytics' );
-	const jetpackVersionSupportsModule = isJetpackMinimumVersion( state, siteId, '4.6-alpha' );
-	const siteSupportsIPAnonymization = siteSupportsGoogleAnalyticsIPAnonymization( state, siteId );
-	const siteSupportsBasicEcommerceTracking = siteSupportsGoogleAnalyticsBasicEcommerceTracking(
-		state,
-		siteId
-	);
-	const siteSupportsEnhancedEcommerceTracking = siteSupportsGoogleAnalyticsEnhancedEcommerceTracking(
-		state,
-		siteId
-	);
 	const siteIsJetpack = isJetpackSite( state, siteId );
 	const googleAnalyticsEnabled =
-		site &&
-		( ! siteIsJetpack || ( siteIsJetpack && jetpackModuleActive && jetpackVersionSupportsModule ) );
+		site && ( ! siteIsJetpack || ( siteIsJetpack && jetpackModuleActive ) );
```

Comment:
nit: can be compressed into `site && ( ! siteIsJetpack || jetpackModuleActive )`

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/30042/commits/361403632eea4a2cb209e7c46133ad6904a0761c 👍